File: 백준/Bronze/1978. 소수 찾기/소수 찾기.java
import java.io.*;
import java.util.*;

public class Main {

    public static void main(String[] args) throws IOException {

        Scanner sc = new Scanner(System.in);

        int t = sc.nextInt();

        int count = 0;
        for (int i = 0; i < t; i++) {
            int number = sc.nextInt();
            if (isPrime(number))
                count++;
        }

        System.out.println(count);
    }

    public static boolean isPrime(int n) {

        if (n < 2)
            return false;

        for (int i = 2; i * i <= n; i++) {
            if (n % i == 0)
                return false;
        }

        return true;
    }
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.cc
#include <iostream>

using namespace std;

bool is_prime(int n);

int main()
{
    int t;
    cin >> t;

    int count = 0;
    while (t--)
    {
        int number;
        cin >> number;

        if (is_prime(number))
            count++;
    }

    cout << count;

    return 0;
}

bool is_prime(int n)
{
    if (n < 2)
        return false;

    for (int i = 2; i * i <= n; i++)
    {
        if (n % i == 0)
            return false;
    }

    return true;
}